README.md

```diff
@@ -1,12 +1,20 @@
 # Weekly roles of the Packit team
 
+![Sharing responsibility in a team](./title.png)
+
 This repository handles the weekly rotating roles of the Packit team.
 Rotating roles represent work that needs to be done each week (e.g. releasing,
-monitoring, communicating with the users). They are described in the issues labelled with `roles`.
+monitoring, communicating with the users). They are described in [issues using the `roles` label](https://github.com/packit/weekly-roles/issues?q=is%3Aissue+is%3Aopen+label%3Aroles).
 These issues are automatically cloned every Friday and team members are assigned to them.
 
 The repository contains:
 
-- `clone_roles.py` - script that does the cloning of the issues, assigning of the team members
-  and closing the old ones
-- `.github/workflows/rotate_roles.yaml` - Github action that runs the script periodically (each Friday)
+1. `clone_roles.py` - script that does the cloning of the issues, assigning of the team members
+   and closing the old ones
+2. `.github/workflows/rotate_roles.yaml` - Github action that runs the script periodically (each Friday)
+
+You can read more about the process in these blog posts:
+
+- [How to run an open-source service?](https://medium.com/@laura.barcziova/how-to-run-an-open-source-service-fb3303240e69)
+- [Inception of rotating roles](https://medium.com/@laura.barcziova/inception-of-rotating-roles-9caf971b3096)
```

Comment:
This link seems broken 🤔

Reply:
![image](https://github.com/packit/weekly-roles/assets/1662493/bf4f8601-fb69-45da-9c67-bbfbde48e4f5)

:confused:

Reply:
will wait with merging once their outage is over

Reply:
Maja, all links work now for me correctly.